assets/css/styles.css

```diff
@@ -321,13 +329,33 @@ header {
   margin: 12px 0;
 }
 
+.project1 {
+  background-image: url("../images/mobile/portfolio1.png");
+  background-repeat: no-repeat;
+}
+
+.project2 {
+  background-image: url("../images/mobile/portfolio2.svg");
+  background-repeat: no-repeat;
+}
+
+.project3 {
+  background-image: url("../images/mobile/portfolio3.svg");
+  background-repeat: no-repeat;
+}
+
+.project4 {
+  background-image: url("../images/mobile/portfolio4.svg");
+  background-repeat: no-repeat;
+}
+
 .card-text {
   display: flex;
```

Comment:
- [ Optional ] You have used this command 93 times, this falls under bad practicing CSS. I would highly recommend you to create a class with this command and use it whenever you need in your HTML.